pkg/process/v6/transformers/osv/transform.go

```diff
@@ -254,7 +254,12 @@ func normalizeFix(fix string, detail *grypeDB.FixDetail) *grypeDB.Fix {
 	}
 }
 
-func normalizeRangeType(t models.RangeType) string {
+func normalizeRangeType(t models.RangeType, ecosystem string) string {
+	// For Bitnami ecosystem, use "bitnami" format instead of "semver"
+	if ecosystem == "Bitnami" && t == models.RangeSemVer {
+		return "bitnami"
```

Comment:
if this ie meant to convey the version format, should we be using these? https://github.com/anchore/grype/blob/ea9d52db9d1b0e6de2beb591fb8245e8465ad570/grype/version/format.go#L22